csrankings-a.csv

```diff
@@ -1649,7 +1649,7 @@ Anitha Gollamudi,University of Massachusetts Lowell,https://scholar.harvard.edu/
 Anja Feldmann [INF],Max Planck Society,https://www.mpi-inf.mpg.de/inet/people/anja-feldmann,mSK3340AAAAJ
 Anja Lehmann,Hasso Plattner Institute,https://hpi.de/das-hpi/personen/professoren/prof-dr-anja-lehmann.html,zH3fwYoAAAAJ
 Anjan Bose,Washington State University,https://vcea.wsu.edu/anjan-bose,jNK8oJoAAAAJ
-Anjan Dutta 0001,University of Exeter,http://emps.exeter.ac.uk/computer-science/staff/ad735,1aKTzmIAAAAJ
+Anjan Dutta 0001,University of Surrey,https://www.surrey.ac.uk/people/anjan-dutta,1aKTzmIAAAAJ

```

Comment:
[Not listed here](https://www.surrey.ac.uk/department-computer-science/people/academic-staff). Please re-open with evidence of how this faculty member meets the inclusion criteria (can solely advise PhD students _in Computer Science_).